tests/unit/api/multiObjectDelete.js

```diff
@@ -185,3 +185,48 @@ describe('getObjMetadataAndDelete function for multiObjectDelete', () => {
             });
     });
```

Comment:
missing tests here, esp. to verify the behaviours which you changed

* Is the cache populated when supported (e.g. call to `getObjectsMD`)
* Is the cache used (e.g. `getObjectMD` not called when object is already in cache)
* Is it working in both "with cache" and "without cache" modes
* Proper "batch" error handling : e.g. verify that we (batch) delete the `data` for each `metadata` we deleted even if there are errors on "other" objects

Reply:
I updated some unit tests: https://github.com/scality/cloudserver/pull/5195/commits/21d22ad1832687ed188f284dcafeaf6ae1c4f285#diff-c2cf43a7212798ef328432b312d2ef420db8c78323dad28d5ea930cdb251b519R227
and added a new test for the last point you mentioned: https://github.com/scality/cloudserver/pull/5195/commits/21d22ad1832687ed188f284dcafeaf6ae1c4f285#diff-c2cf43a7212798ef328432b312d2ef420db8c78323dad28d5ea930cdb251b519R195